ci/infra/libvirt/lb-instances.tf

```diff
@@ -97,7 +97,7 @@ data "template_file" "lb_cloud_init_userdata" {
     username           = var.username
     ntp_servers        = join("\n", formatlist("    - %s", var.ntp_servers))
     hostname           = "${var.stack_name}-lb"
-    hostname_from_dhcp = var.hostname_from_dhcp == true ? "yes" : "no"
+    hostname_from_dhcp = var.hostname_from_dhcp == true && var.cpi_enable == false ? "yes" : "no"
```

Comment:
Maybe we can just use `var.hostname_from_dhcp` instead of `var.hostname_from_dhcp == true`